main.ts

```diff
@@ -446,6 +450,14 @@ class Translator {
         this.visitFunctionLike(funcExpr);
         break;
 
+      case ts.SyntaxKind.MethodSignature:
+        var methodSignatureDecl = <ts.FunctionLikeDeclaration>node;
+        this.emit('abstract');
+        if (methodSignatureDecl.modifiers) this.visitEach(methodSignatureDecl.modifiers);
```

Comment:
Yeah. I'll add a maybeVisitEach for the cases where stuff is nullable. I don't like methods that are too acceptant of their args, so having a strict and a maybe variant might be nice.